contracts/src/PriceFeeds/FXPriceFeed.sol

```diff
@@ -1,50 +1,144 @@
+// SPDX-License-Identifier: MIT
+
 pragma solidity 0.8.24;
 
 import "../Interfaces/IPriceFeed.sol";
 import "../BorrowerOperations.sol";
 
+import { OwnableUpgradeable } from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+
+/**
+ * @title IOracleAdapter
+ * @notice Interface for the Oracle Adapter contract that provides FX rate data
+ */
 interface IOracleAdapter {
```

Comment:
nit: Whilst this interface is only used in the FXPriceFeedContract, I would move this into it's own file so it is consistent with the rest of the codebase.